src/hooks/custom-hooks/useIsGrowthbookLoaded.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import { Analytics } from '@deriv-com/analytics';
+
+const useIsGrowthbookIsLoaded = () => {
+    const [isGBLoaded, setIsGBLoaded] = useState(false);
+
+    useEffect(() => {
+        let checksCounter = 0;
+        const analyticsInterval: NodeJS.Timeout = setInterval(() => {
+            // Check if the analytics instance is available for 10 seconds before setting the feature flag value
+            if (checksCounter > 20) {
```

Comment:
Why do we specifically mention about the interval time? is it 10 or 20?

Reply:
It's 10 seconds.
500ms * 20 (Number of Checks) = 10seconds

Reply:
and we just need to check this for 10 seconds as its not clear when the Growthbook is available on clients machine, and its based on their network speed.